### DISCUSSION TITLE
Allow yfiles_jupyter widget to front end web application

### DISCUSSION THREAD

**Discussion by chanakya009** [yWorks/yfiles-jupyter-graphs-for-neo4j#16] [Q&A]
Did any one tried integrating yfiles_jupyter widget in there any front end web applications

**Comment by yGuy** [comment]
There is no need to "free" the jupyter widget from its jupyter environment to use it in web front end applications. The jupyter widget itself already uses such a widget. So rather than working around the jupyter widget limitations, one should directly use the JavaScript web component library which is used *by* the jupyter widget, directly. The core web widget has no dependencies on Jupyter or python and is specifically meant to be used in "pure" web applications, as it is a client-side-only library.

Check out what is possible, here:

here are almost 300 demos and applications that do not use Jupyter and directly implement web application features:
[demos](https://yfiles.com/demos)

try it yourself in a basic online IDE playground
[playground](https://yfiles.com/play)

or learn more and download the library
[homepage](https://yfiles.com/)
